<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Classes_List</class>
 <widget class="QMainWindow" name="Admin_Classes_List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Classes List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget #centralwidget {
	background-color: #FFFFFF;
}</string>
   </property>
   <widget class="QGroupBox" name="headerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="headerHud">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/header_short.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/back.png</normaloff>:/res/assets/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="avatarLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/cict_avatar.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="adminTitleLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>ADMIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="adminIdLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>45</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <fontweight>Light</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: right;
	font-family: Poppins;
	font-size: 9px;
	font-style: normal;
	font-weight: 300;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Date&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: right;
	font-family: Poppins;
	font-size: 11px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Time&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="footerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>360</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="footerHud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>340</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/footer.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="studentListButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>77</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/home_unselected.png</normaloff>:/res/assets/home_unselected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="studentListLabel">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>102</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>STUDENT LIST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="classListButton">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>77</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/class_list_selected.png</normaloff>:/res/assets/class_list_selected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="classListLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>102</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>CLASS LIST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="whiteBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>130</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/white_box.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="createClassButton">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>25</y>
       <width>210</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/create_class.png</normaloff>:/res/assets/create_class.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="lecturerListButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>77</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/lecturer_unselected.png</normaloff>:/res/assets/lecturer_unselected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lecturerListLabel">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>102</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>LECTURER LIST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>whiteBox</zorder>
    <zorder>footerHud</zorder>
    <zorder>studentListButton</zorder>
    <zorder>studentListLabel</zorder>
    <zorder>classListButton</zorder>
    <zorder>classListLabel</zorder>
    <zorder>createClassButton</zorder>
    <zorder>lecturerListButton</zorder>
    <zorder>lecturerListLabel</zorder>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>110</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #000F2A;
	font-family: Poppins;
	font-size: 13px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
    </property>
    <property name="text">
     <string>CLASSES</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="pageGroup">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>105</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="pageNumberLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>5</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #000F2A;
	text-align: center;
	font-family: Poppins;
	font-size: 13px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pageNextButton">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/next_arrow.png</normaloff>:/res/assets/next_arrow.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pagePrevButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/prev_arrow.png</normaloff>:/res/assets/prev_arrow.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="filterGroup">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>130</y>
      <width>310</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="schoolYearGroup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="schoolYearHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/search_field_short.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QComboBox" name="schoolYearCombobox">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>96</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	background-color: transparent;
	border: 0px;
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 8px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}

QComboBox QAbstractItemView {
    background-color: #000F2A;
}

QComboBox QScrollBar::add-page:vertical,
QComboBox QScrollBar::sub-page:vertical {
    background: none;
	border-style: solid;
}</string>
      </property>
      <property name="placeholderText">
       <string>School Year</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="programGroup">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="programHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/search_field_short.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QComboBox" name="programCombobox">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>96</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	background-color: transparent;
	border: 0px;
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 8px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}

QComboBox QAbstractItemView {
    background-color: #000F2A;
}

QComboBox QScrollBar::add-page:vertical,
QComboBox QScrollBar::sub-page:vertical {
    background: none;
	border-style: solid;
}</string>
      </property>
      <property name="placeholderText">
       <string>Program</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="yearGroup">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="yearHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/search_field_short.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QComboBox" name="yearCombobox">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>96</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	background-color: transparent;
	border: 0px;
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 8px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}

QComboBox QAbstractItemView {
    background-color: #000F2A;
}

QComboBox QScrollBar::add-page:vertical,
QComboBox QScrollBar::sub-page:vertical {
    background: none;
	border-style: solid;
}</string>
      </property>
      <property name="placeholderText">
       <string>Year</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="sectionGroup">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="sectionHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/search_field_short.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QComboBox" name="sectionCombox">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>96</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	background-color: transparent;
	border: 0px;
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 8px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}

QComboBox QAbstractItemView {
    background-color: #000F2A;
}

QComboBox QScrollBar::add-page:vertical,
QComboBox QScrollBar::sub-page:vertical {
    background: none;
	border-style: solid;
}</string>
      </property>
      <property name="placeholderText">
       <string>Section</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="searchGroup">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="searchHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/search_field_short.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchTextbox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>0</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: transparent;
	border: 0px;
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 8px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>5</y>
        <width>15</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/assets/search.png</normaloff>:/res/assets/search.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="semesterGroup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="semesterHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/search_field_short.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QComboBox" name="semesterCombobox">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>96</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	background-color: transparent;
	border: 0px;
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 8px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}

QComboBox QAbstractItemView {
	background-color: #000F2A;
}

QComboBox QScrollBar::add-page:vertical,
QComboBox QScrollBar::sub-page:vertical {
    background: none;
	border-style: solid;
}</string>
      </property>
      <property name="placeholderText">
       <string>Semester</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="classesListArea">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>190</y>
      <width>310</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="contents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>308</width>
       <height>328</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #FFFFFF;
}</string>
     </property>
    </widget>
   </widget>
   <zorder>headerGroup</zorder>
   <zorder>titleLabel</zorder>
   <zorder>pageGroup</zorder>
   <zorder>filterGroup</zorder>
   <zorder>classesListArea</zorder>
   <zorder>footerGroup</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
